src/main/webapp/data/userMapData.json

```diff
@@ -1779,9 +1805,11 @@
       "University of Colorado at Boulder",
       "University of Colorado at Colorado Springs",
```

Comment:
Not related to this issue but it will be easier to change this to `University of Colorado Colorado Springs` here